microsoft-edge/devtools-guide-chromium/changes/changes-tool.md

```diff
@@ -0,0 +1,93 @@
+---
+description: The Changes tool tracks any changes you have made to CSS or JavaScript in DevTools.
+title: Changes tool
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/28/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, changes
+---
+# Changes tool
+
+The **Changes tool** tracks any changes you have made to CSS or JavaScript in DevTools.  Use the **Changes tool** to quickly display all your changes in order to re-apply those changes to the actual source files in your editor (in your Integrated Development Environment; IDE).  This helps you create the necessary changes in your actual source files after you've successfully used DevTools to investigate and fix a problem.
+
+The **Changes tool** is a **Drawer** tool; by default, it opens in the DevTools drawer:
+
+:::image type="content" source="../media/changes-tool/changes-tool-open-reduced.msft.png" alt-text="Developer tools with the Changes tool open in the drawer showing three files that have been modified and the modifications of the selected file." lightbox="../media/changes-tool/changes-tool-open.msft.png":::
+
+In DevTools, use any of the following ways to open the **Changes** tool.  By default, the **Changes** tool appears in the **Drawer**.
+
+
+<!-- ====================================================================== -->
+## Open the Changes tool by right-clicking in a changed file
+
+In the **Sources** tool, right-click any file that shows that it was changed, and then select **Local modifications**:
```

Comment:
@codepo8 - 
How exactly can you make this command appear?  If this command doesn't appear, what should you do to make this command appear?  Under what exact conditions does this **Local modifications** command appear?

How is the page indicated that it was changed?  

What if a page doesn't allow edits, what do you do then?

Reply:
If the page doesn't allow edits, it can never show up as changed. There is an icon overlaying the file icon for changed files, which should be documented in the Sources tool. 

If the file hasn't been touched, local modifications are not an option to even show up.